po/it/system-monitor.po

```diff
@@ -207,7 +207,7 @@ msgid "other"
 msgstr "altro"
 
 msgid "nice"
-msgstr "nice"
```

Comment:
Check [this](http://askubuntu.com/questions/399357/what-does-the-nice-value-mean-in-cpu-utilization-statistics): nice is the time spent on niced (i.e. high priority) tasks. I believe it should not be translated!